### PR TITLE
Feature/714 climate tab wire layers

### PR DESCRIPTION
**Pull request by cschwinderg**
## Related Issues:
* [HMW-726](https://jira.epa.gov/browse/HMW-726)

## Main Changes:
* Finished building out the layers and statuses on the extreme weather tab.
* Split up the "Historical Risk and Potential Future Scenarios" section into two separate sections with the same label.  
  * EPA to provide us with distinct names for these sections. 
* Added disclaimer text and tooltips, discussed in last meeting. There was also an email from Cristina detailing this.
* Moved a lot of the config for this tab to an S3 config file. I might move more stuff, if I have time. 
* Added a county dropdown for viewing data based on any county that intersects with the selected HUC.
  * This dropdown defaults to the county that is displayed on the "Drinking Water" tab.
* Added combined sewer overflows layer, which wasn't in the original mockup.

## Steps To Test:
1. Navigate to http://localhost:3000/community/020802011001/extreme-weather
2. Verify that layers load
3. Verify the switches make the layers visible/hidden
4. Verify that toggling layer visibilty from the layer list also toggles the corresponding switch
5. Block one of the services
6. Verify the status text, displays `N/A`
7. Verify the popups work on various layers
    * Note: Popups do not work on the historical/future Coastal Flooding layer, because it is a TiledImageryLayer
8. Verify the drinking water tab still works

## TODO:
- [ ] Need to get input from EPA on what layers, if any, they want on by default.
- [ ] Need to get input from EPA on naming for the two "Historical Risk and Potential Future Scenarios" sections.
- [ ] Need to get input from EPA on popup content (probably after this is merged and they have time to play with it). The main layers for this are Combined Sewer Overflows, Above/Below Ground Storage Tanks, and Wells. I put in some basic popups for these layers, but EPA may want more info shown there.
- [x] Move more stuff to S3 config. 
- [ ] Research getting popups working on the historical/future Coastal Flooding layer. This layer is a TiledImageryLayer, which doesn't seem to have as good popup support.

### CODE REVIEW

File: app/server/app/public/data/community/extreme-weather.json [hunk 1]
```diff
@@ -0,0 +1,851 @@
...
+              "target": "outline"
+            }
+          ]
+        },
+        "popupTemplate": {
+          "title": "{CountyName}, {StateAbbr}",
+          "content": [
+            {
+              "type": "text",
+              "text": "<p><span style=\"font-size:18px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Maximum number of consecutive dry days</strong></p>"
```
Comment (maxdiebold-erg) [inline app/server/app/public/data/community/extreme-weather.json:222]: This font is a bit larger than size we normally use.
```suggestion
              "text": "<p><span style=\"font-size:14px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Maximum number of consecutive dry days</strong></p>"
```

File: app/server/app/public/data/community/extreme-weather.json [hunk 2]
```diff
@@ -0,0 +1,851 @@
...
+              "target": "outline"
+            }
+          ]
+        },
+        "popupTemplate": {
+          "title": "{CountyName}, {StateAbbr}",
+          "content": [
+            {
+              "type": "text",
+              "text": "<p><span style=\"font-size:18px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Days per year with no precipitation (dry days)</strong></p>"
```
Comment (maxdiebold-erg) [inline app/server/app/public/data/community/extreme-weather.json:330]: Same as above.
```suggestion
              "text": "<p><span style=\"font-size:14px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Days per year with no precipitation (dry days)</strong></p>"
```

File: app/server/app/public/data/community/extreme-weather.json [hunk 3]
```diff
@@ -0,0 +1,851 @@
...
+              "target": "outline"
+            }
+          ]
+        },
+        "popupTemplate": {
+          "title": "{CountyName}, {StateAbbr}",
+          "content": [
+            {
+              "type": "text",
+              "text": "<p><span style=\"font-size:18px;\"><strong>{expression/roundedValue} &nbsp;Inches</strong></span></p><p><strong>Average annual total precipitation</strong></p>"
```
Comment (maxdiebold-erg) [inline app/server/app/public/data/community/extreme-weather.json:438]: Same as above.
```suggestion
              "text": "<p><span style=\"font-size:14px;\"><strong>{expression/roundedValue} &nbsp;Inches</strong></span></p><p><strong>Average annual total precipitation</strong></p>"
```

File: app/server/app/public/data/community/extreme-weather.json [hunk 4]
```diff
@@ -0,0 +1,851 @@
...
+              "target": "outline"
+            }
+          ]
+        },
+        "popupTemplate": {
+          "title": "{CountyName}, {StateAbbr}",
+          "content": [
+            {
+              "type": "text",
+              "text": "<p><span style=\"font-size:18px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Annual days with maximum temperature &gt; 90</strong><span style=\"background-color:rgb(255,255,255);color:rgb(50,50,50);\"><strong>°F</strong></span></p>"
```
Comment (maxdiebold-erg) [inline app/server/app/public/data/community/extreme-weather.json:562]: Same as above.
```suggestion
              "text": "<p><span style=\"font-size:14px;\"><strong>{expression/roundedValue} &nbsp;Days</strong></span></p><p><strong>Annual days with maximum temperature &gt; 90</strong><span style=\"background-color:rgb(255,255,255);color:rgb(50,50,50);\"><strong>°F</strong></span></p>"
```

File: app/client/src/utils/hooks.ts [hunk 5]
```diff
@@ -1570,6 +1575,358 @@ function useSharedLayers({
...
+        {
+          type: 'text',
+          text: '<p style="text-align:center;"><span style="font-size:18px;"><strong>{expression/expr0}</strong></span><br><span style="font-size:14px;">({NIDID})</span></p><p><span style="font-size:18px;">Owner Type: {expression/expr60}</span><br><span style="font-size:18px;">Designed for: {expression/expr16}</span><br><span style="font-size:18px;">Year completed: {expression/expr59}</span><br><span style="font-size:18px;">City: {expression/expr5}</span><br><span style="font-size:18px;">State: {STATE}</span><br><span style="font-size:18px;">Condition Assessment: {CONDITION_ASSESSMENT}</span><br><span style="font-size:18px;">Recent Assessment Date: {CONDITION_ASSESS_DATE}</span></p><p style="text-align:center;"><span style="font-size:18px;"><strong>Specifics</strong></span></p><p><span style="font-size:18px;">Type:&nbsp;{expression/expr13}</span><br><span style="font-size:18px;">Core:&nbsp;{expression/expr14}</span><br><span style="font-size:18px;">Foundation: {expression/expr15}</span><br><span style="font-size:18px;">Dam length: {expression/expr38} ft</span><br><span style="font-size:18px;">Dam height: {expression/expr39} ft</span></p>',
```
Comment (maxdiebold-erg) [inline app/client/src/utils/hooks.ts:1661]: The font size is larger than usual here, too.
```suggestion
          text: '<p style="text-align:center;"><span style="font-size:14px;"><strong>{expression/expr0}</strong></span><br><span style="font-size:14px;">({NIDID})</span></p><p><span style="font-size:14px;">Owner Type: {expression/expr60}</span><br><span style="font-size:14px;">Designed for: {expression/expr16}</span><br><span style="font-size:14px;">Year completed: {expression/expr59}</span><br><span style="font-size:14px;">City: {expression/expr5}</span><br><span style="font-size:14px;">State: {STATE}</span><br><span style="font-size:14px;">Condition Assessment: {CONDITION_ASSESSMENT}</span><br><span style="font-size:14px;">Recent Assessment Date: {CONDITION_ASSESS_DATE}</span></p><p style="text-align:center;"><span style="font-size:14px;"><strong>Specifics</strong></span></p><p><span style="font-size:14px;">Type:&nbsp;{expression/expr13}</span><br><span style="font-size:14px;">Core:&nbsp;{expression/expr14}</span><br><span style="font-size:14px;">Foundation: {expression/expr15}</span><br><span style="font-size:14px;">Dam length: {expression/expr38} ft</span><br><span style="font-size:14px;">Dam height: {expression/expr39} ft</span></p>',
```
Would also be more consistent if the field names are bold.

**Review by maxdiebold-erg — CHANGES_REQUESTED** [review]
The changes look great, and function well. I know some popup content is out of our control, but I did note below some custom ones that could be more consistent with the usual appearance.

**Comment by cschwinderg** [comment]
Good catches! Should be good to go now.

**Review by maxdiebold-erg — APPROVED** [review]

**Pull request merged**